src/game/game.gd

```diff
@@ -0,0 +1,20 @@
+extends Node2D
+
+onready var mapNode: Node2D = $Map
+onready var characterNode: Node2D = $Characters
+onready var rng = RandomNumberGenerator.new()
+
+var characterTemplate: Resource = ResourceLoader.load("res://game/character/character.tscn")
+
+func loadMap(mapPath: String) -> void:
+	var mapToLoad: Node = ResourceLoader.load(mapPath).instance()
+	add_child_below_node(mapNode, mapToLoad)
+
+func addCharacter(networkID) -> void:
```

Comment:
Bit of a nitpick but might as well type `networkID` to be an int, and maybe switch it to be `networkId` for consistency with other scripts but we haven't really decided between `networkId` and `networkID` yet lol